config/custom-environment-variables.json

```diff
@@ -1,4 +0,0 @@
-{
-  "db": "goodpoints_db",
```

Comment:
this is needed file

Reply:
yes for heroku development

Reply:
it's needed for anyone who will clone the project to local machine also. This file shows you what are the variables so that you can overwrite them with ENV variables.  So it needs to stay.

Reply:
ok